src/components/organisms/modals/invitedUserModal.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+
+import { useTranslation } from "react-i18next";
+import { useNavigate } from "react-router-dom";
+
+import { ModalName } from "@enums/components";
+import { OrganizationsService } from "@services";
+import { MemberStatusType } from "@src/enums";
+import { useModalStore, useToastStore, useUserStore } from "@src/store";
+
+import { Button } from "@components/atoms";
+import { Modal } from "@components/molecules";
+
+export const InvitedUserModal = () => {
+	const { t } = useTranslation("modals", { keyPrefix: "invitedUser" });
+	const data = useModalStore((state) => state.data) as { organizationId: string; organizationName: string };
+	const user = useUserStore((state) => state.user);
+	const addToast = useToastStore((state) => state.addToast);
+	const navigate = useNavigate();
+
+	const handleUpdateMemberStatus = async (status: MemberStatusType) => {
+		if (!data?.organizationId || !user?.id) return;
+
+		const { error } = await OrganizationsService.updateMember(data.organizationId, user.id, status);
+		if (error) {
+			addToast({
+				message: t("failedUpdateOrganizationStatus"),
+				type: "error",
+			});
+
+			return;
+		}
+		if (status === MemberStatusType.active) {
+			addToast({
+				message: t("yourResponse"),
+				type: "success",
+			});
+			navigate(`/switch-organization/${data.organizationId}`);
+		}
+	};
```

Comment:
Please move to the userMenu.
In most of our modals (for example all DeleteModals) - the modal is a dummy - it's not doing the action, but his parent doing the action, for example `onDelete`